### DISCUSSION TITLE
libDPS seems nonexistent, yet expected by default

### DISCUSSION THREAD

**Discussion by Pysis868** [ImageMagick/ImageMagick#5742] [Development] [answered]
For the Display PostScript library.

Could someone change this expectation, or suggest where I can find the library, like I just did for libfpx?
Could not find it myself after some light searching.

This makes the former seem more correct: https://bugzilla.redhat.com/show_bug.cgi?id=158984

**Comment by urban-warrior** [comment, marked as answer]
The DPS library is an optional requirement.  The ImageMagick configure script will look for `libdps`, however, when it does not find it, it will continue the build without it.  It's essentially a no-op.  Given so, we certainly could change the default check for the library from `yes` to `no` which would require a user to use the `--with-dps` configure script command-lib argument to search for the library.

**Reply by Pysis868** [reply]
Not required, but currently expected.

That is what I think would be best, especially considering that I can't actually find the library.  Do you know where the source for it is?
